js/stores/appStore.js

```diff
@@ -496,9 +496,10 @@ const handleAppAction = (action) => {
         if (platformUtil.isLinux()) {
           const Channel = require('../../app/channel')
           const desktopName = Channel.getLinuxDesktopName()
+          var args = [desktopName, "--"];
           for (const p of defaultProtocols) {
-            app.setAsDefaultProtocolClient(p, desktopName)
-            app.setAsDefaultProtocolClient('', desktopName)
+            app.setAsDefaultProtocolClient(p, args)
```

Comment:
it should be `setAsDefaultProtocolClient(p, desktopName, ["--"])`, desktopName is not part of args

Reply:
my bad, Linux implementation only reads desktop name so no need to add extra args
the args that needs -- is https://github.com/brave/browser-laptop/pull/12795/files#diff-23ca389e2bcb77191b5a9c10900eb3a3R507

Reply:
https://github.com/brave/muon/blob/80442e401dd3f78d55f78daf2f5f2d8505970296/atom/browser/browser_linux.cc#L92

Reply:
yeah this is not needed for linux afaict